scripts/ci.sh

```diff
@@ -82,8 +83,13 @@ setup_lint() {
 
 setup() {
     echo "Setting up..."
-    update_go
-    setup_protoc
+    if [ ${INIT_GO_PROTOC} -eq 1 ];
+    then
+        update_go
+        setup_protoc
+    else
+        echo "Ignoring Go and Protoc setup"
+    fi
```

Comment:
Can you explain us why we need this pelase?

Reply:
Will write a detailed explanation in the comment below separately.